lib/internal/child_process.js

```diff
@@ -699,6 +700,19 @@ function setupChannel(target, channel, serializationMode) {
     });
   });
 
+  target.on('newListener', function () {
+    if(!target.channel) return;
+
+    const messages = target.channel.pendingMessages;
+    if (!messages.size) return;
+
+    for (const messageParams of messages) {
+      process.nextTick(target.emit.bind(target), ...messageParams);
```

Comment:
We want to avoid spread operator syntax on arrays because it relies on the globally mutable `Array.iterator[Symbol.iterator]` method.
```suggestion
      process.nextTick(() => ReflectApply(target.emit, target, messageParams));
```

Reply:
interesting! Thank you! fixed!